domains/akatsuki.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Miguel-altcell",
+           "email": "wilsonmigueljalla@gmail.com",
+           "discord": "1223721772051075193"
+        },
+    
+        "record": {
+            "CNAME": "akatsuki-crew.rf.gd"
```

Comment:
website is accessible now, but I would like you to add more stuff to the website

Reply:
Yeah im still fixing it. I just started and thank you (:

Reply:
Hey im done now. And i know it looks bad but im still trying to get better